services/py-api/src/database/repository/participants_repository.py

```diff
@@ -1,3 +1,4 @@
+from math import ceil
```

Comment:
You won't need this here so remove it

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4